RadixWallet/Clients/HomeCardsClient/HomeCardsClient+Live.swift

```diff
@@ -0,0 +1,63 @@
+import ComposableArchitecture
+
+// MARK: - HomeCardsClient + DependencyKey
+extension HomeCardsClient: DependencyKey {
+	public static let liveValue: Self = {
+		let observer = HomeCardsObserver()
+
+		// We are hardcoding to `.mainnet` because the cards are currently gateway agnostic. In the future, when Profile is integrated into Sargon, it will be Sargon
+		// observing the current gateway and defining the networkId to use.
+		let manager = HomeCardsManager(networkAntenna: URLSession.shared, networkId: .mainnet, cardsStorage: HomeCardsStorage(), observer: observer)
+
+		return Self(
+			cards: {
+				observer.subject.eraseToAnyAsyncSequence()
+			},
+			removeCard: { card in
+				Task {
+					try? await manager.cardDismissed(card: card)
+				}
+			},
+			walletStarted: {
+				Task {
+					try? await manager.walletStarted()
+				}
+			},
+			walletCreated: {
+				Task {
+					try? await manager.walletCreated()
+				}
+			},
+			deepLinkReceived: { value in
+				Task {
+					try? await manager.deepLinkReceived(encodedValue: value)
+				}
+			}
+		)
+	}()
+}
+
+// MARK: - HomeCardsManager + Sendable
+extension HomeCardsManager: @unchecked Sendable {}
+
+// MARK: - HomeCardsStorage
+private final class HomeCardsStorage: Sargon.HomeCardsStorage {
+	@Dependency(\.userDefaults) var userDefaults
```

Comment:
just making sure, will this be using `UserDefaults.Dependency.radix`? You can check the actual values are displayed in Debug Settings

Reply:
all good. It wasn't showing any data but I've updated it so that it either shows `Data available` or `No Data available` for home cards key.